client/src/helpers/APICalls/createRequestNotification.ts

```diff
@@ -0,0 +1,29 @@
+interface FetchOptions {
+  method: string;
+  headers?: {
+    'Content-Type': string;
+  };
+  body?: string;
+}
+
+const createRequestNotification = async (name: string, duration: number, recipient: string) => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      type: 'serviceRequest',
+      title: 'dogSitting',
+      description: `${name} has requested your service for ${duration} hours.`,
+      date: new Date(),
+      recipient: recipient,
+    }),
+  };
```

Comment:
Looks solid 💯 My only suggestion would be to indicate that `recipient` field expects an id by renaming it (e.g. `recipientId` or `recipientProfileId`)